app/views/events/index.html.erb

```diff
@@ -25,6 +25,7 @@
 
 <%= link_to 'New Event', new_event_path %>
 
+
 <table>
```

Comment:
It might be worth pulling this table erb stuff into its own partial as it looks like you're reusing the logic for past events.